src/linear_algebra/affine_space/affine_equiv.lean

```diff
@@ -300,13 +300,31 @@ def const_vsub (p : P₁) : P₁ ≃ᵃ[k] V₁ :=
 
 variable (P₁)
 
-/-- The map `p ↦ v +ᵥ p` as an affine automorphism of an affine space. -/
-@[simps]
+/-- The map `p ↦ v +ᵥ p` as an affine automorphism of an affine space.
+
+Note that there is no need for an `affine_map.const_vadd` as it is always an equivalence.
```

Comment:
This is obvious, but it means people searching for `affine_map.const_vadd` get a result!